readme.md

```diff
@@ -190,6 +197,62 @@ The example above, when applied to a project rooted at `~/project`, would create
 Note that repeated calls to `assets` are processed in-order, and as such, it is possible to overwrite resources
 by specifying that a later invocation be relocated to a previously used destination's ancestor directory.
 
+### Pod Distribution plugin
+
+Apply the plugin using standard Gradle convention:
+
+    plugins {
+        id 'com.palantir.sls-pod-distribution'
+    }
+
+A sample configuration for the Pod plugin:
+
+    distribution {
+        serviceName "pod-name"
```

Comment:
What about changing this method to `podName`? It just reads a bit weirdly when you're defining a pod not a service!